config/crossSections.yaml

```diff
@@ -0,0 +1,748 @@
+  unc: 0.0001955
+DYJetsToLL_M-800to1000:
+  crossSec: 0.03406
+  reference: XSDB DYJetsToLL_M-800to1000_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2020-09-22 01:32:34
+  unc: 0.0001636
+DYJetsToLL_M-1000to1500:
+  crossSec: 0.01828
+  reference: XSDB DYJetsToLL_M-1000to1500_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2020-09-22 01:32:19
+  unc: 8.743e-05
+DYJetsToLL_M-1500to2000:
+  crossSec: 0.002367
+  reference: XSDB DYJetsToLL_M-1500to2000_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2020-09-22 01:32:23
+  unc: 1.143e-05
+DYJetsToLL_M-2000to3000:
+  crossSec: 0.0005409
+  reference: XSDB DYJetsToLL_M-2000to3000_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2020-09-22 01:32:27
+  unc: 2.745e-06
+DYJetsToLL_0J_MLL_200_400:
+  crossSec:
+  reference: XSDB
+  unc:
+DYJetsToLL_0J_MLL_400_800:
+  crossSec:
+  reference: XSDB
+  unc:
+DYJetsToLL_0J_MLL_800_1400:
+  crossSec: 0.02492
+  reference: XSDB DYJetsToLL_0J_MLL_800_1400_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2022-08-13 06:43:21
+  unc: 6.134e-05
+DYJetsToLL_0J_MLL_1400_2300:
+  crossSec: 0.001792
+  reference: XSDB DYJetsToLL_0J_MLL_1400_2300_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2022-08-13 06:43:19
+  unc: 4.57e-06
+DYJetsToLL_0J_MLL_2300_3500:
+  crossSec:
+  reference: XSDB
+  unc:
+DYJetsToLL_0J_MLL_3500_4500:
+  crossSec: 3.979e-06
+  reference: XSDB DYJetsToLL_0J_MLL_3500_4500_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2022-08-13 06:43:19"
+  unc: 1.052e-08
+DYJetsToLL_0J_MLL_4500_6000:
+  crossSec: 4.007e-07
+  reference: XSDB DYJetsToLL_0J_MLL_4500_6000_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2022-08-13 06:43:20
+  unc: 1.499e-09
+DYJetsToLL_0J_MLL_6000_Inf:
+  crossSec: 2.788e-08
+  reference: XSDB DYJetsToLL_0J_MLL_6000_Inf_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2022-08-13 06:43:20
+  unc: 1.042e-10
+DYJetsToLL_1J_MLL_200_400:
+  crossSec: 2.22
+  reference: XSDB DYJetsToLL_1J_MLL_200_400_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2022-08-13 14:23:27
+  unc: 0.04284
+DYJetsToLL_1J_MLL_400_800:
+  crossSec: 0.2417
+  reference: XSDB DYJetsToLL_1J_MLL_400_800_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2022-08-13 06:43:23
+  unc: 0.001478
+DYJetsToLL_1J_MLL_800_1400:
+  crossSec: 0.01836
+  reference: XSDB DYJetsToLL_1J_MLL_800_1400_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2022-08-13 06:43:23
+  unc: 0.0001037
+DYJetsToLL_1J_MLL_1400_2300:
+  crossSec: 0.001509
+  reference: XSDB DYJetsToLL_1J_MLL_1400_2300_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2022-08-13 06:43:21
+  unc: 1.326e-05
+DYJetsToLL_1J_MLL_2300_3500:
+  crossSec: 8.822e-05
+  reference: XSDB DYJetsToLL_1J_MLL_2300_3500_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2022-08-13 06:43:22
+  unc: 4.414e-07
+DYJetsToLL_1J_MLL_3500_4500:
+  crossSec: 3.432e-06
+  reference: XSDB DYJetsToLL_1J_MLL_3500_4500_TuneCP5_13TeV-amcatnloFXFX-pythia8 modifiedOn 2022-08-13 06:43:22
+  unc: 1.69e-08
+DYJetsToLL_1J_MLL_4500_6000:
+  crossSec: 4.007e-07
+  reference: XSDB
```

Comment:
put process name & modification date as for the other samples

Reply:
ok